pkg/cmd/issue/edit/edit.go

```diff
@@ -32,6 +32,7 @@ type EditOptions struct {
 	Interactive  bool
 
 	prShared.Editable
+	RemoveMilestone bool
```

Comment:
Does this need to be added to `gh pr edit` command in order to fulfill the PR title?

https://github.com/cli/cli/blob/42da8baf7046b801e4c9c78a23effc048a2c81c5/pkg/cmd/pr/edit/edit.go#L19-L33

Reply:
Yeah. I'll update the `gh pr edit` command.